.github/workflows/release-code-freeze.yml

```
@@ -143,6 +144,63 @@ jobs:
           git checkout trunk
           git checkout -b ${{ needs.prepare-for-feature-freeze.outputs.nextReleaseBranch }}
           git push origin ${{ needs.prepare-for-feature-freeze.outputs.nextReleaseBranch }}
+      - name: Create milestone for next dev cycle
+        uses: actions/github-script@v7
+        with:
+          script: |
```

actionlint (1.7.9)

property "prepare-for-code-freeze" is not defined in object type {prepare-for-feature-freeze: {outputs: {nextreleasebranch: string; nextreleaseversion: string; nexttrunkmainversion: string}; result: string}}

(expression)